roles/rebuild/tasks/validate.yml

```diff
@@ -0,0 +1,16 @@
+- block:
+  - stat:
+      path: "{{ openhpc_rebuild_clouds }}"
+    register: openhpc_clouds_path
+  - name: Check OpenRC file (openhpc_rebuild_clouds) exists
+    assert:
+      that: openhpc_clouds_path.stat.exists
+      fail_msg: "openrc file {{ openhpc_rebuild_clouds }} on localhost (specified by `openhpc_rebuild_clouds`) does not exist"
+  - name: Check OpenRC file (openhpc_rebuild_clouds) is usable
+    shell: |
```

Comment:
Add no_log here to avoid leaking a valid token.

Reply:
See here: https://docs.ansible.com/ansible/latest/reference_appendices/logging.html#protecting-sensitive-data-with-no-log